src/graphql/query/appealBanner.graphql

```diff
@@ -1,10 +1,29 @@
 query {
 	general {
-		setting(setting: "feature.endofyear_appeal.enabled") {
+		appeal_enabled: setting(setting: "feature.endofyear_appeal.enabled") {
```

Comment:
Aliases for all settings and checks against their start and end time.

Reply:
Nice, it's great we can pass through and check the group vs all the individual settings.